go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/code-ready/crc
 go 1.13
 
 require (
-	github.com/BurntSushi/toml v0.3.1 // indirect
 	github.com/Masterminds/semver v1.5.0
```

Comment:
Fwiw, given their size, I'd suggest moving the go.mod/go.sum/vendor updates to a separate commit. They do not really make sense without the newly added code, but this would make the commits easier to look at/ignore.

Reply:
done :+1: